engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -397,6 +398,9 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
       } else {
         goto(WorkflowExecutionAbortedState)
       }
+    case Event(EngineStatsActor.JobCountQuery, data) =>
+      sender ! EngineStatsActor.JobCount(data.backendJobExecutionActors.size)
```

Comment:
Actually I think this might be a case where the value in the FSM data is a misnomer. I remember being a little confused by that the other day. it's actually an EJEA